catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/SecureHadoopCatalogOperations.java

```diff
@@ -231,6 +227,12 @@ public Fileset alterFileset(NameIdentifier ident, FilesetChange... changes)
       if (userInfo != null) {
         userInfoMap.put(NameIdentifier.of(ident.namespace(), finalName), userInfo);
       }
+
+      Boolean impersonationEnable = impersonationEnableMap.remove(ident);
```

Comment:
It's already weird for us to have userInfoMap. Do we need impersonationEnableMap? Could we give a unified abstraction data type for them.

Reply:
Do you have any good suggestions? Using a map is just a way to calculate the real UGI and impersonation value by name identifier. What we can do is just combine them into one and use a class to hold these two map, I don't see any improvement or refinement for it, if you have better idea, please let me know, thanks.